crates/swc_ecma_transforms_typescript/src/inline_enum.rs

```diff
@@ -0,0 +1,144 @@
+use std::{cell::RefCell, rc::Rc};
+
+use swc_atoms::JsWord;
+use swc_common::collections::AHashMap;
+use swc_ecma_ast::{
+    AssignExpr, AssignPat, AssignPatProp, ComputedPropName, Expr, Id, Lit, MemberExpr, MemberProp,
+    Module, Script, Str, Tpl,
+};
+use swc_ecma_utils::ident::IdentLike;
+use swc_ecma_visit::{as_folder, noop_visit_mut_type, Fold, VisitMut, VisitMutWith};
+
+use crate::TSEnumConfig;
+
+pub(crate) type TSEnumLit = Rc<RefCell<AHashMap<Id, AHashMap<JsWord, Lit>>>>;
+
+pub fn inline_enum(ts_enum_lit: TSEnumLit, ts_enum_config: TSEnumConfig) -> impl Fold + VisitMut {
```

Comment:
This function is exposing a crate-local type (`TSEnumLit`) in public api.

Reply:
Maybe it should be `pub(crate) fn`?

Reply:
Yes, `inline_enum` should not be public API